src/uu/ls/src/ls.rs

```diff
@@ -2663,9 +2676,14 @@ fn display_item_long(
         output_display += "  ";
     }
     if let Some(md) = item.get_metadata(out) {
+        #[cfg(not(all(unix, not(any(target_os = "android", target_os = "macos")))))]
```

Comment:
I don't like the double negation, maybe it's better to remove the entire `cfg` and use variable shadowing.
```suggestion
```

Reply:
great idea :)

Reply:
Looks like my suggestion was wrong :|

Reply:
no worries, I simplified the expression